engine/kubernetes/engine.config-map.yml

```diff
@@ -8,10 +8,10 @@ data:
   HOST: http://localhost:8080
   ENVIRONMENT: development
   LOG_LEVEL: debug
-  DATABASE_URL: sqlite:///engine.db
```

Comment:
Maybe keep the option to use SQLite instead of PostgreSQL by commenting the SQLite option? So the user can choose